lib/rules/web/admin_console/4.10/logs.xyaml

```diff
@@ -13,15 +13,15 @@ switch_to_container:
 wait_log_window_loaded:
   elements:
   - selector:
-      xpath: //div[contains(@class,'log-window')]/div[contains(@class,'log-window__header')]
+      xpath: //div[contains(@class,'log-window__header')]
     timeout: 30
   - selector:
-      xpath: //div[contains(@class,'log-window')]/div[contains(@class,'log-window__body')]
+      xpath: //div[contains(@class,'pf-c-log-viewer__main')]
     timeout: 30
 check_log_content_contains:
   element:
     selector:
-      xpath: //div[contains(@class,'log-window__lines') and contains(.,'<log_content>')]
+      xpath: //div[contains(@class,'pf-c-log-viewer__list-item') and contains(.,'<log_content>')]
```

Comment:
Is it `//span[contains(@class, 'text') and contains(.,'<log_content>')]` ?

Reply:
yes, should use span